runtime/lib/backend/lightning/lightning_dynamic/LightningSimulator.cpp

```diff
@@ -115,6 +115,9 @@ auto LightningSimulator::One() const -> Result
 void LightningSimulator::NamedOperation(const std::string &name, const std::vector<double> &params,
                                         const std::vector<QubitIdType> &wires, bool inverse)
 {
+    // Although ISWAP and PSWAP exist in Lightning::simulator_gate_info, they are only supported on
```

Comment:
The list of `simulator_gate_info` is there to check the number of parameters and wires for each requested gate on  simulators. This array is not specific to Lightning simulators. Users should be able to pass any gate name and backend devices should provide neat error messages for unsupported gates. This is currently missing for the Lightning simulator, but I ensure this  will be added upstream.

Reply:
You can remove line 118-120.

Reply:
done